ksp_bridge/src/ksp_bridge.cpp

```diff
@@ -6,7 +6,9 @@ KSPBridge::KSPBridge()
     : rclcpp::Node("ksp_bridge")
 {
     declare_parameter<int64_t>("update_interval", 10);
-    declare_parameter<std::vector<std::string>>("celestial_bodies", {});
+    printf("Declaring parameter: %s\n", "celestial_bodies"  );
```

Comment:
Why are you outputting this? Also `%s` expects a `const char*` not an `<std::vector<std::string>`.